need to have some mechanism to prevent from memory corruption

studentBC opened the pull request.

studentBC committed:
need to have some mechanism to prevent from memory corruption

chiahaol commented on StitchPanorama.py:
```diff
@@ -1,77 +1,118 @@
+    def prepare_lists(self):
+        self.centerIdx = self.count/2
+        self.center_im = self.images[int(self.centerIdx)]
+        for i in range(self.count):
```
I think there should be a way to avoid using for loop, like self.images[: self.centerIndex]

studentBC replied:
but every frame is uniq one so you still need a for loop within it like self.images[frames[i], for i in range(self.centerIndex)]

chiahaol replied:
self.left_list = self.images[:self.centerIndex + 1]

studentBC committed:
finish panorama stitching

Both of these two solutions for stitch left and right will casue memory corrupt
when we have too many frames to deal with

chiahaol commented on StitchPanorama.py:
```diff
@@ -1,77 +1,123 @@
-            return 0, img 
+    def leftshift(self):
+        a = self.left_list[0]
+        for b in self.left_list[1:]:
```
I think this line will create a copy, it's better to use index to iterate
